app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -40,13 +40,33 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               label="Title"
               name="title"
               className="w-full flex-grow-0"
+              placeholder="Enter note title"
               required
             />
             <Textarea
               label="Description"
               name="description"
               className="w-full flex-grow-0"
-              rows={8}
+              rows={1}
+              placeholder="Enter note description"
+              required
+            />
+            <Select
+              label="Assigned Contact"
+              name="contact"
+              className="w-full flex-grow-0"
+              isClearable
+              options={NAMES.map(name => ({ label: name, value: name }))}
+              placeholder="Select Role"
+              required
```

Comment:
Keep the implicit `true` attrs to the top of the component. [Refer](https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Also%20note%20that%20these%20attributes%20ought%20to%20be%20the%20first%20among%20the%20list%20of%20attributes%20being%20passed%20down%20as%20props%20because%20it%20gives%20a%20better%20visual%20hierarchy%20and%20also%20these%20are%20the%20props%20that%20don%E2%80%99t%20change%20often)